docs/interfaces.md

```diff
@@ -460,6 +460,13 @@ processes on the system.
 
 * Auto-Connect: no
 
+### system-power-control
```

Comment:
Okay, let's go with `shutdown` then. That keeps `reboot` available for us to offer as a less powerful feature (one that just asks snapd to reboot when possible).